data/model/autoprune.py

```diff
@@ -509,11 +508,14 @@ def delete_autoprune_task(task):
 
 
 def prune_tags(tags, repo, namespace):
+    # imported here to avoid cyclic imports
+    from data.logs_model import logs_model
```

Comment:
Not a huge fan of this fix, this is a patch that increases complexity and may lead to maintainability issues later on